rules/windows/process_creation/win_susp_ping_hex_ip.yml

```diff
@@ -15,9 +15,9 @@ logsource:
     product: windows
 detection:
     selection:
-        CommandLine:
-            - '*\ping.exe 0x*'
-            - '*\ping 0x*'
+        CommandLine|contains:
+            - '\ping.exe 0x'
+            - '\ping 0x'
```

Comment:
better detect ping as a ProcessName (there is such executable)

Reply:
Fixed